downstream/assemblies/platform/assembly-appendix-inventory-file-vars.adoc

```diff
@@ -1,4 +1,4 @@
-[id="appendix-inventory-files-vars"]
+[id="assembly-appendix-inventory-files-vars"]
```

Comment:
```suggestion
[id="appendix-inventory-files-vars"]
```
There are references to this ID in the docset, so we need to leave the ID as `appendix-inventory-files-vars`

Reply:
Thank you for the suggestion, Aine. The doc builds correctly now.